package.json

```diff
@@ -126,13 +126,13 @@
     "rollup-plugin-license": "^3.0.1",
     "sass": "^1.43.4",
     "sass-loader": "^12.3.0",
-    "stylelint": "^15.11.0",
-    "stylelint-config-recommended-scss": "^12.0.0",
-    "stylelint-no-unsupported-browser-features": "^7.0.0",
-    "stylelint-order": "^6.0.3",
-    "stylelint-prettier": "^4.1.0",
-    "stylelint-scss": "^5.0.1",
-    "stylelint-use-logical": "^2.1.0",
+    "stylelint": "^16.6.1",
+    "stylelint-config-recommended-scss": "^14.0.0",
+    "stylelint-no-unsupported-browser-features": "^8.0.1",
```

Comment:
Yay, finally no more false alarms on `position: sticky` and other obviously supported features